arch/arm/core/aarch32/cortex_m/fault.c

```diff
@@ -710,14 +711,14 @@ static inline bool z_arm_is_synchronous_svc(z_arch_esf_t *esf)
 	uint16_t fault_insn = *(ret_addr - 1);
 #else
 	SCB->CCR |= SCB_CCR_BFHFNMIGN_Msk;
-	__DSB();
-	__ISB();
+	barrier_data_synchronization_fence_full();
+	barrier_instruction_synchronization_fence_full();
```

Comment:
Would ...

```suggestion
	barrier_dsync_fence_full();
	barrier_isync_fence_full();
```

and `barrier_dmem_fence_full`(etc.) be more concise but still clear?

Reply:
yup, sounds fine to me. I'll change that.

Reply:
+1 to shorter names

Reply:
done